app/assets/stylesheets/_preferences.scss

```diff
@@ -44,3 +45,38 @@
     @extend .color-info;
   }
 }
+
```

Comment:
If we continued with the zero-client-CSS approach of utility-first classes, all of these should either belong on StyleKit as more generic reusable classes or on the HTML markup, without using extend

Reply:
I've removed the component classes. Created separate components and used utility classes as an interim solution until we decide.